compiler/src/dmd/toir.d

```diff
@@ -346,6 +346,7 @@ elem *getEthis(const ref Loc loc, IRState *irs, Dsymbol fd, Dsymbol fdp = null,
         {
             fd.error(loc, "is a nested function and cannot be accessed from `%s`", irs.getFunc().toPrettyChars());
             return el_long(TYnptr, 0); // error recovery
+            // assert(0);
```

Comment:
why is this commented?

Reply:
It's not; it was spurious. I've removed it now.